Third iteration (#5)
clearhaus/3DSv2-api-documentation: source/getting-started.rst

Getting Started
###############

This page is intented to familiarize you with superficial parts of the 3-D
Secure v2 and to provide you with *some* understanding of how a 3-D Secure v2

> authentication flow looks.

Reviewer: `s/how (a 3-D Secure v2 authentication flow) looks/\1/`

Author: Absolutely correct :+1:

Authentication
==============

In 3-D Secure v2, cardholder authentication might not necessarily involve the
cardholder.

> A authentication flow that does not involve the cardholder is called a

Reviewer: Perhaps "An authentication flow not involving the cardholder".

Author: :+1:

> *frictionless flow*, an flow that involves the cardholder is called a

Reviewer: `s/, an/, whereas a/`

Reviewer: Nah. Perhaps ", whereas a flow involving the cardholder".

Author: I like it :+1:

*challenge flow*.
The *frictionless flow* utilizes device fingerprinting to verify cardholder

> involvement and involves fewer steps than a *challenge flow*.

Reviewer 2: ```suggestion
involvement and reduces the steps needed compared to the *challenge flow*.
```

Author: How about:

> The *frictionless flow* utilizes device fingerprinting to verify cardholder
> authenticity, providing a smoother flow for the cardholder.

The idea of frictionsless is to not involve the cardholder to provide a smoother flow.

Reviewer 2: Fine by me :-)

parameter definition. Brief descriptions are:

/preauth
  This is used when performing transactions from a browser (as opposed to using

>   an SDK), where it will return an optional 3-D Secure Method URL, which is

Reviewer 2: ```suggestion
  This is used when performing transactions from a browser as opposed to using
  an SDK. It will return an optional 3-D Secure Method URL, which is
```

Author: :+1:

  used for browser fingerprinting. This can support risk-based analysis and
  assist in ensuring a flow where the cardholder is not challenged.

  While in the transition period between 3-D Secure v1 and v2, this endpoint
  can help determine if v1 should be used instead. This is documented in the
  :ref:`3ds_versioning` guide.

/auth
  The primary API method to provide authentication data to the issuer.  Under
  certain circumstances, the authentication flow will end successfully here,

>   this is called *frictionless* flow.

Reviewer: Perhaps "here. This is called a"

8. The :ref:`postauth-endpoint` is used to fetch the results of the
   authentication.
9. Nominally a ``RReq`` is returned to the Requestor. Parameters are detailed
   in the :ref:`postauth response <postauth-response>` section.

Sandbox environment
===================

> A sandbox environment is included to ease integration. It is

Reviewer: Perhaps `s/ease/facilitate/`

> A sandbox environment is included to ease integration. It is

Reviewer: Perhaps `s/ease/facilitate/`

Reviewer: Nah. Perhaps "A sandbox environment is provided as a service for initial integration, continuous integration and live tests."

Author: > A sandbox environment is included as a service for initial and continuous
> integration, and for live tests. This is our implementation so discrepancies
> with ACS implementations are unavoidable.
